easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.10-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2019.10'
+versionsuffix = '-Python-3.7.4'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+dependencies = [
+    ('Python', '3.7.4'),
+]
+
+use_pip = True
+
+# order is important!
+exts_list = [
+    ('numpy', '1.17.3', {
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'source_urls': ['https://pypi.python.org/packages/source/n/numpy/'],
+        'checksums': ['a0678793096205a4d784bd99f32803ba8100f639cf3b932dc63b21621390ea7e'],
+    }),
+    ('scipy', '1.3.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/scipy/'],
+        'checksums': ['2643cfb46d97b7797d1dbdb6f3c23fe3402904e3c90e6facfe6a9b98d808c1b5'],
+    }),
+    ('mpi4py', '3.0.2', {
+        'source_urls': ['https://bitbucket.org/mpi4py/mpi4py/downloads/'],
+        'checksums': ['f8d629d1e3e3b7b89cb99d0e3bc5505e76cc42089829807950d5c56606ed48e0'],
+    }),
+    ('pandas', '0.25.2', {
```

Comment:
@furstj Let's bump this to 0.25.3 which was released Nov 1st?

Other than this, I think this PR is good to go...